### PR TITLE
Display QRCode at the end of the gd games export flow

### PR DESCRIPTION
**Pull request by AlexandreSi**
https://gdevelop-storybook.s3.amazonaws.com/qrcode-on-publish-to-gdgames/latest/index.html?path=/story/share-onlinegamelink--done-with-private-build
https://gdevelop-storybook.s3.amazonaws.com/qrcode-on-publish-to-gdgames/latest/index.html?path=/story/share-onlinegamelink--done-with-public-build

### CODE REVIEW

File: newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js [hunk 1]
```diff
@@ -303,37 +304,44 @@ const OnlineGameLink = ({
...
-                        hideLoader
-                        suggestAdditionalActions
-                      />
-                    </ColumnStackLayout>
-                  )}
-                  {!isBuildPublished && game && (
+                  <ShareLink url={buildOrGameUrl} />
+                  <ColumnStackLayout noMargin expand>
+                    {navigator.share ? (
+                      <ShareButton url={buildOrGameUrl} />
```
Comment (ClementPasteau) [inline newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js:318]: You changed the logic a bit here to display the share buttons even for instant builds.
Is that on purpose?
I think it's fine, just checking

**Review by ClementPasteau — APPROVED** [review]
maybe share a screenshot of the result?

File: newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js:318]: Yes, I did it on purpose: even if the link is private, you might want to send it to someone else or yourself

**Comment by AlexandreSi** [comment]
I attached the links to the stories of interest

**Comment by AlexandreSi** [comment]
> Can the QRcode be bigger? I have to put my phone 10 cm from the screen because of the dpi of my screen, I think on a retina screen it's crisper so it shouldn't be a problem :/

I increased the size from 100px to 150px on non-mobile devices, it should work better

**Pull request closed without merging**